Source/Atomic/Scene/Node.cpp

```diff
@@ -958,6 +958,32 @@ void Node::GetChildren(PODVector<Node*>& dest, bool recursive) const
         GetChildrenRecursive(dest);
 }
 
+void Node::GetChildrenWithName(PODVector<Node*>& dest, const String& name, bool recursive) const
+{
+	GetChildrenWithName(dest, StringHash(name), recursive);
+}
+
+void Node::GetChildrenWithName(PODVector<Node*>& dest, const char* name, bool recursive) const

```

Comment:
Just a note, I believe this overload exists to resolve ambiguity between the String& and StringHash overloads when passing a quoted string, as it chooses String, I don't think we need the StringHash nameHash overload at all as this is searching by name which shouldn't be hashed (like a type's name)